Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/GyrometerSamplePage.xaml

```diff
@@ -0,0 +1,81 @@
+﻿<Page x:Class="Uno.Gallery.Views.Samples.GyrometerSamplePage"
+	  xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	  xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	  xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	  xmlns:smtx="using:ShowMeTheXAML"
+	  xmlns:local="using:Uno.Gallery"
+	  xmlns:muxc="using:Microsoft.UI.Xaml.Controls"
+	  mc:Ignorable="d">
+
+	<Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+		<local:SamplePageLayout IsDesignAgnostic="True">
+			<local:SamplePageLayout.DesignAgnosticTemplate>
+				<DataTemplate>
+					<StackPanel>
+						<smtx:XamlDisplay UniqueKey="GyrometerSamplePage_Sample"
+										  smtx:XamlDisplayExtensions.IgnorePath="XX"
```

Comment:
invalid ignore path?
you can use "XamlDisplay\Grid" or dont this property

Reply:
I will remove this line completely. In the case of non-ui sample I don't think it is useful to show the xaml implementation.

Reply:
regardlessly set or not, you are still displaying the xaml when using the XamlDisplay control
IgnorePath just trim down the amount of xaml code displayed

but, you still want to show the c# code

Reply:
Oh right! And if I want to show only the c# code? Setting the IgnorePath to XX worked for me correctly.